src/openslide-decode-dicom.c

```diff
@@ -33,73 +33,89 @@ void _openslide_dicom_propagate_error(GError **err, DcmError *dcm_error) {
   dcm_error_clear(&dcm_error);
 }
 
+// implements DcmIO
+struct _dicom_io {
```

Comment:
Nit: this is awkwardly placed between the two error-handling functions.  Let's move it above `_openslide_dicom_propagate_error`?